browser/ui/views/split_view/split_view_separator.cc

```diff
@@ -184,6 +184,14 @@ void SplitViewSeparator::OnBoundsChanged(const gfx::Rect& previous_bounds) {
   LayoutMenuButton();
 }
 
+void SplitViewSeparator::OnPaintBackground(gfx::Canvas* canvas) {
+  auto* cp = GetColorProvider();
```

Comment:
Not really sure why this was working before - we didn't seem to be setting the background

Reply:
I assume that separator's bg color comes frome its parent?

Reply:
It's parent was a widget, and it was see through before this change. Maybe that bit came from upstream?

Reply:
(it was fine in Beta, not in Nightly, but that could be related to Nala)

Reply:
`kColorBraveSplitViewMenuButtonBackground` is that color. it fills that that widget's bg.

Reply:
oh, wait, this seems not related with menu button?

Reply:
Yeah, it's the divider line, not the button :smile: